tests/integration/WC_Facebook_Product_Test.php

```diff
@@ -31,6 +31,122 @@ protected function _after() {
 	/** Test methods **************************************************************************************************/
 
 
+	/** @see \WC_Facebook_Product::prepare_product() */
+	public function test_prepare_product_not_ready_for_commerce_gender() {
```

Comment:
Can't these 3 tests be one?

Reply:
They could, but we should avoid multiple asserts in a single integration test method where possible

Reply:
Whoops, looks like I forgot to link: https://github.com/skyverge/lumiere/wiki/Best-practices#single-asserts--logic

That’s from back when we first started defining this stuff though, ages ago, so we can certainly amend. Thoughts?

Reply:
I think it's a tradeoff. Sometimes I believe it's worth having multiple asserts, if it would make the tests a bit faster and would not add a lot of complexity. 🤷‍♀️